### PR TITLE
Drones shut down, rather than gib, with a Robotics Console kill order

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
balance: Kill orders from the Robotics Console on drones cause the drone's body to shut down, rather than self destruct.
del: Syndicate drones no longer start at 30/120 hp, instead, they have
30/30 hp.
/:cl:

- Makes randomising polymorphed drones a var
- Changes the types of free drones to make stuff clearer
- To be honest, the "hacked" var is a bit shit, since we don't ever have
law modifications of drones, even though we maintain the hypothetical
code basis for it. Probably should phase it out.

A Robotics Console drone can still be repaired by someone with a screwdriver, if they feel that the drone's shut down order was illegitimate.

### CODE REVIEW

**Comment by TehZombehz** [comment]
But then somebody might screwdriver the drone and revive them, which would negate my free one-click griff valid!

**Comment by KorPhaeron** [comment]
Seems odd that its harder to kill dronew than cyborgs

**Comment by Jalleo** [comment]
Have you removed the drone can eat drone to heal mechanic?

**Comment by coiax** [comment]
No, that still exists.

**Comment by Dorsisdwarf** [comment]
why are you so obsessed with stopping syndie drones from eating NT drones

**Comment by coiax** [comment]
Because they can just repair themselves to 120 hp with a screwdriver, and they can kill syndi drones and eat them.

**Comment by ChangelingRain** [comment]
They don't actually start at 30 HP because they have no damage, any damage they take will reset them to 120 minus whatever the damage was.

**Comment by coiax** [comment]
They're simple animals, I thought they used the health var

**Comment by ChangelingRain** [comment]
they use the bruteloss ver for damage

**Comment by Shadowlight213** [comment]
@Coiax can you add logging to blowing up drones by chance?
Atm it's unlogged I think.

**Comment by coiax** [comment]
@Shadowlight213 done.

**Comment by BeeSting12** [comment]
Isn't there no way to remove a drone's duffel? If he stole something this is the only way to get it back

**Comment by coiax** [comment]
Explaining to the drone that they stole something is often very effective. Otherwise, I don't think there's any method of getting the dufflebag, they don't have a drag menu, I don't think

File: code/game/machinery/computer/robot.dm [hunk 1]
```diff
@@ -159,14 +158,17 @@
...
-				s.set_up(3, 1, D)
-				s.start()
-				D.visible_message("<span class='danger'>\the [D] self destructs!</span>")
-				D.gib()
+				var/turf/T = get_turf(D)
+				do_sparks(3, TRUE, D)
+				D.visible_message("<span class='danger'>[D] shuts down!</span>", "<span class='userdanger'>Your body automatically shuts down, in response to a kill code.</span>")
+				D.death()
+
+				message_admins("[ADMIN_LOOKUPFLW(usr)] shut down [key_name(D, D.client)][ADMIN_JMP(T)]!")
```
Comment (Cyberboss) [inline code/game/machinery/computer/robot.dm]: key_name_admin(D)

**Review by Cyberboss — APPROVED** [review]

**Comment by optimumtact** [comment]
Just add a lockdown mode for drones and leave the destruction button alone

:-1:

**Comment by coiax** [comment]
In what circumstances do you lock down a drone? You can't put DNM on a PR because you personally dislike it, fruit.

**Comment by coiax** [comment]
@tgstation/commit-access please control your fruit's tagging

**Comment by optimumtact** [comment]
So just so we're clear, you change the console kill order on drones to be effectively the same as a lockdown (execpt you can revive the drone with a screwdriver) and yet somehow I'm the biased one for suggesting you just add a lockdown here?

**Comment by Shadowlight213** [comment]
@optimumtact borgs can't simply ghost out of being locked down.
The pr kills the drone. Lockdown doesn't kill a borg.
Why add a lockdown to a role that can just ghost out and grab another one?

**Comment by optimumtact** [comment]
It's effectively the same as it locks the drone in place and then can be revived with a screwdriver.

You are arguing semantics

**Comment by Shadowlight213** [comment]
What? No. A locked borg is alive. It can talk, call for help, etc. A dead drone can't talk to its friends.
Borgs also are stuck in one spot and can't just jump out of their body and observe.
Are you suggesting we make lockdown actually kill the borg? Make it a special form of dead so it can't be used for free revives?

**Comment by lzimann** [comment]
keep drones gibbing

**Comment by Repukan** [comment]
Bad change.

Stop trying to grudge code Joan only gets away with it because it's a whole game mode not a ghost role.

**Comment by Supermichael777** [comment]
drones are silicons because hands

**Comment by ExcessiveUseOfCobblestone** [comment]
Why are these 2 very different things not atomized? I don't like either change.

despite my reoccurring dislike about some of the aspects of ghost roles on lavaland, I think having a role that base builds in lavaland [like drone but not obnoxious] would be nice if you preferred building with impunity.

**Comment by lzimann** [comment]
@KorPhaeron Thoughts? Otherwise I'll probably just close.

**Comment by Shadowlight213** [comment]
Cobby has a point. Atomize the health change from the console one

**Comment by lzimann** [comment]
Guess no thoughts on this then.

**Pull request closed without merging**